modules/andes-core/broker/src/main/java/org/wso2/andes/store/cache/GuavaBasedMessageCacheImpl.java

```diff
@@ -190,17 +201,13 @@ public AndesMessagePart getContentFromCache(long messageId, int offsetValue) {
         AndesMessage cachedMessage = getMessageFromCache(messageId);
         AndesMessagePart part = null;
         if (null != cachedMessage) {
-            part = cachedMessage.getContentChunkList().get(offsetValue);
-            if (null == part) {
-                for (AndesMessagePart currentPart : cachedMessage.getContentChunkList()) {
-                    if (currentPart.getOffSet() == offsetValue) {
-                        part = currentPart;
-                    }
-                }
-            }
+            //the offset comes as a multiple of DEFAULT_CONTENT_CHUNK_SIZE after being converted in method
+            //'getMessageContentChunkConvertedCorrectly' in {@link org.wso2.andes.amqp.AMQPUtils}
```

Comment:
could we change 'getMessageContentChunkConvertedCorrectly' to something meaningful word 'correctly' seems odd!